x-pack/auditbeat/auditbeat.yml

```diff
@@ -47,11 +47,16 @@ auditbeat.modules:
   - /usr/sbin
   - /etc
 
+- module: system
+  datasets:
+    - package # Installed, updated, and removed packages
+
+  period: 30m # The frequency at which the datasets check for changes
```

Comment:
I think we should find a compromise between the original 10s and 30m. We don't want to overload the system but at the same time we need to be responsive to catch a package that is installed for a short time. What about 2-3 minutes? @andrewkroh what do you think?

Another improvement to consider in the future would be monitor some arch-specific file for changes and then trigger the enumeration of packages.

Reply:
I'll be waiting for the final default value

Reply:
`2m` sounds good to me.

Reply:
:+1: